tests/integration/docusaurus/connecting_to_your_data/fluent_datasources/how_to_connect_to_data_on_s3_using_pandas.py

```diff
@@ -0,0 +1,69 @@
+"""
+To run this code as a local test, use the following console command:
+```
+pytest -v --docs-tests -m integration -k "how_to_connect_to_data_on_s3_using_pandas" tests/integration/test_script_runner.py
+```
+"""
+
+# Python
+# <snippet name="tests/integration/docusaurus/connecting_to_your_data/fluent_datasources/how_to_connect_to_data_on_s3_using_pandas.py get_context">
+import great_expectations as gx
+
+context = gx.get_context()
+# </snippet>
+
+# Python
+# <snippet name="tests/integration/docusaurus/connecting_to_your_data/fluent_datasources/how_to_connect_to_data_on_s3_using_pandas.py define_add_pandas_s3_args">
+datasource_name = "my_s3_datasource"
+bucket_name = "my_bucket"
+boto3_options = {}
+# </snippet>
+
+bucket_name = "superconductive-docs-test"
+
+# Python
+# <snippet name="tests/integration/docusaurus/connecting_to_your_data/fluent_datasources/how_to_connect_to_data_on_s3_using_pandas.py create_datasource">
+datasource = context.sources.add_pandas_s3(
+    name=datasource_name, bucket=bucket_name, boto3_options=boto3_options
+)
+# </snippet>
+
+assert datasource_name in context.datasources
+
+# Python
```

Comment:
Does commenting python here add syntax highlighting?

Reply:
@TrangPham Good question -- I am not sure.  It may not be necessary, because in the document, there is a "python" tag.  However, this has been the style I inherited from others' examples and found it to be aesthetically nice.  Maybe we can try locally using `invoke docs` and see whether or not it adds anything to the HTML view.  Thanks.